src/v1.js

```diff
@@ -30,6 +30,7 @@ var api = new API({
     'https://docs.taskcluster.net/reference/core/taskcluster-hooks/docs/firing-hooks',
     'for more information.',
   ].join('\n'),
+  name: 'HooksAPI',
```

Comment:
Hm, maybe we should document this better (/cc @imbstack) but this needs to be the lower-case name of the service, without the "taskcluster-" prefix, so in this case, "hooks".  I'll push a fix to this branch and merge.